core/src/main/java/org/apache/xtable/delta/DeltaConversionSource.java

```diff
@@ -109,7 +109,7 @@ public InternalSnapshot getCurrentSnapshot() {
   @Override
   public TableChange getTableChangeForCommit(Long versionNumber) {
     InternalTable tableAtVersion = tableExtractor.table(deltaLog, tableName, versionNumber);
-    // Client to call getCommitsBacklog and call this method.
+    // Source to call getCommitsBacklog and call this method.
```

Comment:
Unrelated to this PR: this comment is not clear to me. Do you think more clarity is needed here?

Reply:
I don't understand this comment, may be leftover from some dev work so I am just removing it.